### PR TITLE
Picker button fix

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
blackbaud/skyux-theme#139

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix)

File: src/app/public/modules/datepicker/datepicker.component.scss [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 .sky-input-group-datepicker-btn {
```
Comment (Blackbaud-PaulCrowder) [inline src/app/public/modules/datepicker/datepicker.component.scss:3]: Does this class even need CSS rules now? Or does the button still have a border radius if not specified? If it does, that should probably be fixed in theme.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix)

File: src/app/public/modules/datepicker/datepicker.component.scss (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/datepicker/datepicker.component.scss]: Upgraded theme library. This line has now been removed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-picker-button-fix)

**Pull request merged**